packages/p2p/src/utils/orders.ts

```diff
@@ -336,26 +534,44 @@ export default class ExtendedOrderDetails {
         };
     }
 
+    /**
+     * Function that returns the user details of the order
+     *
+     * @returns {object} user details
+     */
     get my_user_details() {
         return this.is_my_ad ? this.order_details.advertiser_details : this.order_details.client_details;
     }
```

Comment:
I don't think we are using this. Can you please check and remove if not being used.